scripts/cross-framework-test/README.md

```diff
@@ -0,0 +1,87 @@
+# Cross-Framework Build Tester
+
+A cross-framework build testing script for thorvg.web that tests build functionality across different frontend frameworks.
+
+## File Structure
+
+All paths are relative to the **project root directory**:
+
+```
+scripts/cross-framework-test/
+├── index.js              # Main entry point
+├── constant/
+│   └── config.js             # Config and framework settings
+├── lib/
+│   ├── build-tester.js       # Main build testing logic
+│   ├── cli.js                # Command line interface
+│   ├── file-system.js        # File system utilities
+│   ├── logger.js             # Logging utility
+│   └── reporter.js           # Results formatting and output
+└── README.md             # This file
```

Comment:
@tinyjin 
Hello, I’ve drafted a proposal for issue [140](https://github.com/thorvg/thorvg.web/issues/140). The diff is a bit large, but I believe that’s mainly because configuration files are included. If further clarification is needed, I’ll be glad to add comments. Please review with a focus on (1) the example setup and (2) the script structure. Thank you.